### PR TITLE
Feature/newinteractive list

### PR DESCRIPTION
**Pull request by tnyandoro**
Description
In this project, you will add some functionality to your application to make it interactive. The user will be able to reorder the list by dragging each item and moving it into the desired position. The user will also be able to mark task completion by selecting the corresponding checkbox (or undo it by unchecking the checkbox). The updated tasks list will be stored in local storage.

IMPORTANT NOTE: Read all requirements before you start building your project.

General requirements
Make sure that there are no linter errors.
Make sure that you used correct GitHub Flow.
Make sure that you documented your work in a professional way.
HTML/CSS requirements
Follow our list of the best practices for HTML & CSS.
JavaScript requirements
Follow our list of best practices for JavaScript.
Project requirements
Add 2 new JavaScript files and import them as modules:
one will contain methods related to drag / drop and sorting functionality.
the other will contain methods related to the status updates (completed: true / false).
Add event listeners to list items (dragstart, dragover, drop) and to checkbox (change).
Update items object's values for index and completed keys upon user actions.
Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

File: src/ItemRepository.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+export default class ItemRepository {
+  constructor() {
+    this.todoItems = [];
+    this.loadItems();
+  }
+
```
Comment (carlos-ssh) [inline src/ItemRepository.js:6]: - Kindly add the [right linters](https://github.com/microverseinc/linters-config/tree/master/html-css-js) into your project.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @tnyandoro, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Your project, it's really nice. I like the description in your pull request, your README file and the requirements.


## Required Changes ♻️
_Check the comments under the review._ 👇🏼👇🏼👇🏼👇🏼

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tnyandoro** [comment]
-  Correct Linters used and fixed

**Review by carlos-ssh — APPROVED** [review]
Hi @tnyandoro , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**